2021-08-24 01:45:08,495 INFO    MainThread:6288 [wandb_setup.py:_flush():69] setting env: {}
2021-08-24 01:45:08,496 INFO    MainThread:6288 [wandb_setup.py:_flush():69] setting login settings: {}
2021-08-24 01:45:08,497 INFO    MainThread:6288 [wandb_init.py:_log_setup():342] Logging user logs to F:\5th sem project\wandb\run-20210824_014508-3anu7bws\logs\debug.log
2021-08-24 01:45:08,497 INFO    MainThread:6288 [wandb_init.py:_log_setup():343] Logging internal logs to F:\5th sem project\wandb\run-20210824_014508-3anu7bws\logs\debug-internal.log
2021-08-24 01:45:08,499 INFO    MainThread:6288 [wandb_init.py:_jupyter_setup():294] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x000002BC8F577340>
2021-08-24 01:45:08,519 INFO    MainThread:6288 [wandb_init.py:init():375] calling init triggers
2021-08-24 01:45:08,519 INFO    MainThread:6288 [wandb_init.py:init():380] wandb.init called with sweep_config: {}
config: {}
2021-08-24 01:45:08,519 INFO    MainThread:6288 [wandb_init.py:init():395] re-initializing run, found existing run on stack: 21s9e8mc
2021-08-24 01:45:08,519 INFO    MainThread:6288 [wandb_run.py:finish():1254] finishing run khushal/weather-recognization/21s9e8mc
2021-08-24 01:45:08,520 INFO    MainThread:6288 [wandb_run.py:_atexit_cleanup():1554] got exitcode: 0
2021-08-24 01:45:08,521 INFO    MainThread:6288 [wandb_run.py:_restore():1526] restore
2021-08-24 01:45:08,919 INFO    MainThread:6288 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 476
  total_bytes: 476
}

2021-08-24 01:45:09,081 INFO    MainThread:6288 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 476
  total_bytes: 476
}

2021-08-24 01:45:09,896 INFO    MainThread:6288 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 476
  total_bytes: 476
}

2021-08-24 01:45:10,945 INFO    MainThread:6288 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 476
  total_bytes: 19375
}

2021-08-24 01:45:11,048 INFO    MainThread:6288 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 476
  total_bytes: 19375
}

2021-08-24 01:45:11,152 INFO    MainThread:6288 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 476
  total_bytes: 19375
}

2021-08-24 01:45:11,256 INFO    MainThread:6288 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 476
  total_bytes: 19375
}

2021-08-24 01:45:11,359 INFO    MainThread:6288 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 476
  total_bytes: 19375
}

2021-08-24 01:45:11,469 INFO    MainThread:6288 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 476
  total_bytes: 19375
}

2021-08-24 01:45:11,580 INFO    MainThread:6288 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 476
  total_bytes: 19375
}

2021-08-24 01:45:11,684 INFO    MainThread:6288 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 476
  total_bytes: 19375
}

2021-08-24 01:45:11,787 INFO    MainThread:6288 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 476
  total_bytes: 19375
}

2021-08-24 01:45:11,907 INFO    MainThread:6288 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 476
  total_bytes: 19375
}

2021-08-24 01:45:12,011 INFO    MainThread:6288 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 476
  total_bytes: 19375
}

2021-08-24 01:45:12,114 INFO    MainThread:6288 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 476
  total_bytes: 19375
}

2021-08-24 01:45:12,218 INFO    MainThread:6288 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 476
  total_bytes: 19375
}

2021-08-24 01:45:12,321 INFO    MainThread:6288 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 19375
  total_bytes: 19375
}

2021-08-24 01:45:12,427 INFO    MainThread:6288 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 19375
  total_bytes: 19375
}

2021-08-24 01:45:12,530 INFO    MainThread:6288 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 19375
  total_bytes: 19375
}

2021-08-24 01:45:12,635 INFO    MainThread:6288 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 19375
  total_bytes: 19375
}

2021-08-24 01:45:12,737 INFO    MainThread:6288 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 19375
  total_bytes: 19375
}

2021-08-24 01:45:12,842 INFO    MainThread:6288 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 19375
  total_bytes: 19375
}

2021-08-24 01:45:12,945 INFO    MainThread:6288 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 19375
  total_bytes: 19375
}

2021-08-24 01:45:13,881 INFO    MainThread:6288 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 19375
  total_bytes: 19375
}

2021-08-24 01:45:13,985 INFO    MainThread:6288 [wandb_run.py:_wait_for_finish():1676] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 19375
  total_bytes: 19375
}

2021-08-24 01:45:15,535 INFO    MainThread:6288 [wandb_run.py:_show_files():1898] logging synced files
2021-08-24 01:45:15,547 INFO    MainThread:6288 [wandb_init.py:init():424] starting backend
2021-08-24 01:45:15,547 INFO    MainThread:6288 [backend.py:_multiprocessing_setup():70] multiprocessing start_methods=spawn, using: spawn
2021-08-24 01:45:15,555 INFO    MainThread:6288 [backend.py:ensure_launched():135] starting backend process...
2021-08-24 01:45:15,736 INFO    MainThread:6288 [backend.py:ensure_launched():139] started backend process with pid: 13300
2021-08-24 01:45:15,737 INFO    MainThread:6288 [wandb_init.py:init():429] backend started and connected
2021-08-24 01:45:15,739 INFO    MainThread:6288 [wandb_run.py:_label_probe_notebook():798] probe notebook
2021-08-24 01:45:15,739 INFO    MainThread:6288 [wandb_run.py:_label_probe_notebook():805] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2021-08-24 01:45:15,739 INFO    MainThread:6288 [wandb_init.py:init():477] updated telemetry
2021-08-24 01:45:15,741 INFO    MainThread:6288 [wandb_init.py:init():500] communicating current version
2021-08-24 01:45:20,344 INFO    MainThread:6288 [wandb_init.py:init():505] got version response 
2021-08-24 01:45:20,345 INFO    MainThread:6288 [wandb_init.py:init():513] communicating run to backend with 30 second timeout
2021-08-24 01:45:21,160 INFO    MainThread:6288 [wandb_init.py:init():540] starting run threads in backend
2021-08-24 01:45:26,188 INFO    MainThread:6288 [wandb_run.py:_console_start():1584] atexit reg
2021-08-24 01:45:26,192 INFO    MainThread:6288 [wandb_run.py:_redirect():1458] redirect: SettingsConsole.WRAP
2021-08-24 01:45:26,193 INFO    MainThread:6288 [wandb_run.py:_redirect():1495] Wrapping output streams.
2021-08-24 01:45:26,198 INFO    MainThread:6288 [wandb_run.py:_redirect():1519] Redirects installed.
2021-08-24 01:45:26,198 INFO    MainThread:6288 [wandb_init.py:init():565] run started, returning control to user process
2021-08-24 01:45:26,226 INFO    MainThread:6288 [wandb_setup.py:_flush():69] setting env: {}
2021-08-24 01:45:26,227 INFO    MainThread:6288 [wandb_setup.py:_flush():69] setting login settings: {}
2021-08-24 01:45:26,227 INFO    MainThread:6288 [wandb_init.py:_log_setup():342] Logging user logs to F:\5th sem project\wandb\run-20210824_014526-2gakv4dp\logs\debug.log
2021-08-24 01:45:26,228 INFO    MainThread:6288 [wandb_init.py:_log_setup():343] Logging internal logs to F:\5th sem project\wandb\run-20210824_014526-2gakv4dp\logs\debug-internal.log
2021-08-24 01:45:26,229 INFO    MainThread:6288 [wandb_init.py:init():375] calling init triggers
2021-08-24 01:45:26,230 INFO    MainThread:6288 [wandb_init.py:init():380] wandb.init called with sweep_config: {}
config: {}
2021-08-24 01:45:26,230 INFO    MainThread:6288 [wandb_init.py:init():395] re-initializing run, found existing run on stack: 3anu7bws
2021-08-24 01:45:26,231 INFO    MainThread:6288 [wandb_run.py:finish():1254] finishing run khushal/whether reconizer/3anu7bws
